test/parser/parser_test.cpp

```diff
@@ -341,6 +341,12 @@ TEST_F(ParserTestBase, InsertTest) {
   EXPECT_EQ(insert_stmt->GetInsertionTable()->GetTableName(), "foo");
   EXPECT_EQ(insert_stmt->GetInsertColumns()->size(), 3);
   EXPECT_EQ((*insert_stmt->GetValues())[0][0]->GetExpressionType(), ExpressionType::VALUE_DEFAULT);
+
+  result = parser::PostgresParser::BuildParseTree("INSERT INTO foo VALUES (1, 1+1, 1+2);");
+  insert_stmt = result->GetStatement(0).CastManagedPointerTo<InsertStatement>();
+  EXPECT_EQ(insert_stmt->GetInsertionTable()->GetTableName(), "foo");
+  EXPECT_EQ(insert_stmt->GetInsertColumns()->size(), 0);
+  EXPECT_EQ((*insert_stmt->GetValues())[0][1]->GetExpressionType(), ExpressionType::OPERATOR_PLUS);
```

Comment:
It would be nice to check that the numbers are correct too, but I think this is alright.